src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/dependency/GerritDependencyAction.java

```diff
@@ -0,0 +1,44 @@
+package com.sonyericsson.hudson.plugins.gerrit.trigger.dependency;
+
+import hudson.EnvVars;
+import hudson.model.AbstractBuild;
+import hudson.model.EnvironmentContributingAction;
+import hudson.model.InvisibleAction;
+import hudson.model.Run;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Adds Action that stores the data about dependency jobs.
+ */
+public class GerritDependencyAction extends InvisibleAction implements EnvironmentContributingAction {
+    private List<Run> runs = Collections.emptyList();
```

Comment:
If this action is ever serialized to disk anywhere, which I suppose it is (seems to be added to `queue.xml` and `EnvironmentContributingAction` is added to `build.xml), then this field is illegal. It will in fact explode on you: https://github.com/jenkinsci/jenkins/pull/2957

Maybe you just want a `List<String>` of `id`s?